saleor/graphql/order/types.py

```diff
@@ -0,0 +1,33 @@
+import graphene
+from graphene import relay
+from graphene_django import DjangoObjectType
+
+from ...order import models
+from ..core.types import CountableDjangoObjectType
+
+
+class Order(CountableDjangoObjectType):
+    order_id = graphene.Int(
+        description='Order number.')
+
+    class Meta:
+        description = 'Represents an order in the shop.'
+        interfaces = [relay.Node]
+        model = models.Order
+        exclude_fields = []
```

Comment:
We should exclude `total_net` and `total_gross` in favor of having only `total` field to access both values. The same applies to the `shipping_price` fied. `TaxedMoney` type should also give access to `tax` property.